src/main/java/ch/uzh/ifi/hase/soprafs23/controller/UserController.java

```diff
@@ -345,6 +344,6 @@ public ResponseEntity<Void> setReady(@PathVariable Long userId, @PathVariable Lo
             userService.setAllUsersNotReady(memberIds);
         }
 
-        return new ResponseEntity<>(HttpStatus.OK);
+        return new ResponseEntity<>(HttpStatus.OK); //TODO returns 200 or 204?
```

Comment:
imo should be 204 no content

Reply:
@OrestisBollano  this is your part originally, can I change it?